spec/jira/resource/attachment_spec.rb

```diff
@@ -71,7 +71,7 @@
       issue = JIRA::Resource::Issue.new(client)
       path_to_file = './spec/mock_responses/issue.json'
       attachment = JIRA::Resource::Attachment.new(client, issue: issue)
-      attachment.save!('file' => path_to_file)
+      attachment.save!('file' => path_to_file, mimeType: 'image/jpeg')
```

Comment:
Have you looked into the `mime-types` gem? With it you can do something like 

```
MIME::Types.type_for(filename).first.content_type
```

to get the type from the extension rather than hardcoding it.